tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/NativeMobile/MemoryAndStorage/Supplier Get Mobile Memory And Storage.feature

```diff
@@ -11,21 +11,20 @@ Background:
         | SolutionId | SolutionName   | SupplierStatusId | SupplierId |
         | Sln1       | MedicOnline    | 1                | Sup 1      |
         | Sln2       | TakeTheRedPill | 1                | Sup 1      |
-        | Sln3       | PracticeMgr    | 1                | Sup 1      |
     And solutions have the following details
         | SolutionId | SummaryDescription             | FullDescription   | ClientApplication                                                                                    |
         | Sln1       | An full online medicine system | Online medicine 1 | { "MobileMemoryAndStorage" : { "MinimumMemoryRequirement": "1GB", "Description": "A description" } } |
-        | Sln3       | Testing System                 | Full System       | {  }                                                                                                 |
+        | Sln2       | Testing System                 | Full System       | {  }                                                                                                 |
 
 @3607
 Scenario: Mobile Memory And Storage are retrieved for the solution
     When a GET request is made for native-mobile-memory-and-storage section for solution Sln1
     Then a successful response is returned
     And the string value of element minimum-memory-requirement is 1GB
     And the string value of element storage-requirements-description is A description
-
+    
```

Comment:
Can you remove the additional spaces added here?

Reply:
Removed addition line